xgboost-classifier/xgboost-softmax.R

```diff
@@ -0,0 +1,109 @@
+# Use SigOpt to tune an XGBoost softmax classifier in R
+# Learn more about SigOpt's R Client:
+# https://sigopt.com/docs/overview/R
```

Comment:
There was a change recently that broke https://sigopt.com/docs/overview/R
(not sure if we're planning on making it work again, cc @pfhayes)
You'll have to use lower-case r: https://sigopt.com/docs/overview/r